[CO-2860] Updated /users/export/ids documentation to reflect the new rate limit / request param (#7968)
braze-inc/braze-docs: _docs/_api/endpoints/export/user_data/post_users_identifier.md

  "braze_id": (optional, string) Braze identifier for a particular user,
  "email_address": (optional, string) Email address of user,
  "phone": (optional, string) Phone number of user,

>   "fields_to_export": (required, array of strings) Name of user data fields to export.

Reviewer: ```suggestion
  "fields_to_export": (required, array of strings) Name of user data fields to export
```

>   "fields_to_export": (required, array of strings) Name of user data fields to export. Defaults to all if not provided

Reviewer 2: Probably want to get rid of "Defaults to all...", right?

Author: Yea good point,  updated

}
```

{% alert note %}

> For customers who onboarded with Braze on or after August 22, 2024. Request parameter `fields_to_export` will be required.

Reviewer: ```suggestion
For customers who have onboarded with Braze on or after August 22, 2024, the request parameter `fields_to_export` is required.
```
